deploy/chart/crds/0000_50_olm_00-catalogsources.crd.yaml

```diff
@@ -79,6 +79,9 @@ spec:
                     priorityClassName:
                       description: If specified, indicates the pod's priority. If not specified, the pod priority will be default or zero if there is no default.
                       type: string
+                    securityContextConfig:
+                      description: 'SecurityContextConfig can be one of `legacy` or `restricted`. The CatalogSource''s pod is either injected with the right pod.spec.securityContext and pod.spec.container[*].securityContext values to allow the pod to run in Pod Security Admission(PSA) controller''s `restricted` mode, or doesn''t set these values at all so that the pod can be run in PSA `privileged` namespace. By default, SecurityContextConfig is set to `restricted`. If a value other than `legacy` or `restricted` is used as input, the default value is set instead of using the incorrect value When using older catalog images, which could not be run in restricted mode, the SecurityContextConfig should be set to `legacy`. More info about PSA can be found here: https://kubernetes.io/docs/concepts/security/pod-security-admission/'
```

Comment:
setting a value other than `restricited` or `legacy` should be an error, not fallback to the default.

leaving it empty should fallback to the default.

Reply:
```suggestion
                      description: 'SecurityContextConfig can be one of `legacy` or `restricted`. The CatalogSource''s pod is either injected with the right pod.spec.securityContext and pod.spec.container[*].securityContext values to allow the pod to run in Pod Security Admission(PSA) controller''s `restricted` mode, or doesn''t set these values at all in which case the pod can only run in PSA `privileged` namespaces. By default, SecurityContextConfig is set to `restricted`. If the value is unspecified, the default value of `restricted` is used.  Specifying any other value will result in a validation error.  When using older catalog images, which could not be run in restricted mode, the SecurityContextConfig should be set to `legacy`. More information about PSA can be found here: https://kubernetes.io/docs/concepts/security/pod-security-admission/'
```